adapter-cqld4/src/main/resources/activities/baselines/cql-iot-dse.yaml

```diff
@@ -45,63 +41,39 @@ blocks:
          'compaction_window_unit': 'MINUTES',
          'split_during_flush': true
         };
-       tags:
-         name: create-table
-     - truncate-table: |
-         truncate table <<keyspace:baselines>>.<<table:iot>>;
-       tags:
-         name: truncate-table
-  - tags:
-      phase: rampup
+
+      truncate-table: |
+        truncate table <<keyspace:baselines>>.<<table:iot>>;
+  rampup:
     params:
       cl: <<write_cl:LOCAL_QUORUM>>
-    statements:
-     - insert-rampup: |
-        insert into  <<keyspace:baselines>>.<<table:iot>>
-        (machine_id, sensor_name, time, sensor_value, station_id, data)
-        values ({machine_id}, {sensor_name}, {time}, {sensor_value}, {station_id}, {data})
-        using timestamp {cell_timestamp}
-       idempotent: true
-       tags:
-         name: insert-rampup
-  - tags:
-      phase: verify
-      type: read
+      idempotent: true
+    ops:
+      rampup-insert: |
+        insert into  <<keyspace:baselines>>.<<table:iot>> (machine_id, sensor_name, time, sensor_value, station_id, data) values ({machine_id}, {sensor_name}, {time}, {sensor_value}, {station_id}, {data}) using timestamp {cell_timestamp};
```

Comment:
These long lines should be wrapped and block indented to make this easier to read, edit in the wild.